### ISSUE TITLE
Improve links between admins

### ISSUE THREAD

**Issue by wagnerand** [mozilla/addons-server#20235]
On the add-on detail admin, we have a `Files` section, but no way to access the version admin. The add-on detail admin should show a list of versions, along with the version id, file id, creation date of the version, version number, channel, file status (dropdown as is currently), block status, deleted (read-only), needs_human_review (read-write). 

The version id and file id columns should link to the respective admin detail pages.

**Comment by diox** [comment]
@wagnerand Have you ever used the `Recalc Hash` link btw ? I think these days it's completely obsolete.

**Comment by wagnerand** [comment]
> @wagnerand Have you ever used the `Recalc Hash` link btw ? I think these days it's completely obsolete.

Not in years, it's also why I didn't list it above. I think we can remove that functionality, especially if it's obsolete.

**Comment by diox** [comment]
> needs_human_review (read-write)

Because that flag lives on the version, not the file, it's a bit annoying to allow modifying it there. Is that shortcut worth it for you ? It's already possible to modify it on the version admin, and we'd link to that. I'd rather not build code specifically for this small use case if we can avoid it.

**Comment by wagnerand** [comment]
> > needs_human_review (read-write)
> 
> Because that flag lives on the version, not the file, it's a bit annoying to allow modifying it there. Is that shortcut worth it for you ? It's already possible to modify it on the version admin, and we'd link to that. I'd rather not build code specifically for this small use case if we can avoid it.

Ok, let's omit that for now, then. I thought it would be the same complexity as the file status, which we already support changing here.

**Comment by diox** [comment]
Yeah unfortunately status is already on the file model so we get it for free, but for versions fields we need extra specific work. In the glorious distant future where the 2 models are merged this won't be a problem...

**Comment by diox** [comment]
Behold my UX sk1llz:
![Screenshot 2023-01-25 at 13-33-59 115 casaur-gultur-raccock Change addon Django site admin](https://user-images.githubusercontent.com/187006/214564583-cb009d5c-e969-454e-9e42-bc959864a7f5.png)

**Comment by AlexandraMoga** [comment]
I've verified this issue on AMO -dev and the requested fields/functionalities have been added to the addon admin model:

![image](https://user-images.githubusercontent.com/31961530/215486606-d2490169-8e28-437f-bc4a-2c1b2542616b.png)

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDSRV-224
